recipes/create-dmg/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class CreateDmgConan(ConanFile):
+    name = "create-dmg"
+    description = "A shell script to build fancy DMGs"
+    license = "MIT"
+    topics = "command-line", "dmg"
+    homepage = "https://github.com/create-dmg/create-dmg"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def validate(self):
+        if self.settings.os != "Macos":
+            raise ConanInvalidConfiguration(f"{self.name} works only on MacOS")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy("create-dmg", dst="bin", src=self._source_subfolder)
+
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+
```

Comment:
```suggestion

    def package_id(self):
        del self.info.settings.build_type
        del self.info.settings.compiler
        del self.info.settings.arch

```
remove settings here.